docs/content/development/extensions-core/namespaced-lookup.md

```diff
@@ -0,0 +1,320 @@
+---
+layout: doc_page
+---
+
+<div class="note caution">
+This extension has been deprecated as of 0.9.1.
+</div>
+
+# Transitioning to lookups-cached-global
+
+New users should reference the exteion [lookups-cached-global](lookups-cached-global.html)
+
+## Moving to cluster wide configuration
```

Comment:
I think we should link this doc from lookups.md and lookups-cached-global.md, with language like "If you are currently using the druid-namespace-lookup extension, please see namespaced-lookup.html for instructions on migrating to the new lookup framework. The druid-namespace-lookup extension has been deprecated and will be removed in a future version of Druid."